src/Features/VisualBasic/Portable/Completion/CompletionProviders/SymbolCompletionProvider.ItemRules.vb

```diff
@@ -23,6 +23,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
                 Return MyBase.IsCommitCharacter(completionItem, ch, textTypedSoFar)
             End Function
 
+            Public Overrides Function IsBetterPreselectedMatch(item As CompletionItem, other As CompletionItem, textTypedSoFar As String) As Boolean?
+                Dim symbolItem = DirectCast(item, SymbolCompletionItem)
+                Dim otherSymbolItem = DirectCast(other, SymbolCompletionItem)
+                If TypeOf otherSymbolItem.Symbols.First() Is ITypeSymbol Then
+                    Return symbolItem.Symbols.First().MatchesKind(SymbolKind.Event, SymbolKind.Field, SymbolKind.Local, SymbolKind.Method,
```

Comment:
To my other comment: should we encode this as two different priorities of "expression" versus "declaration" or something else, rather than having two priority systems active here?